.ci/cico_deploy.sh

```diff
@@ -68,21 +73,23 @@ docker push "${PROD_IMAGE_DEVSHIFT_LATEST}"
 
 echo "CHE VALIDATION: Image pushed to devshift registry"
 
-# We need to continue pushing to the Docker Hub until we have setup a webhook for 
-# repository che/che on devshift. The webhook should trigger
-# https://jenkins.cd.test.fabric8.io/che-version-updater/notify 
-# every time a new version of Che is available 
-PROD_IMAGE_DOCKER_HUB="rhche/rh-che-server:${CHE_SERVER_DOCKER_IMAGE_TAG}"
+if [ $TARGET != "rhel" ]; then
+    # We need to continue pushing to the Docker Hub until we have setup a webhook for
+    # repository che/che on devshift. The webhook should trigger
+    # https://jenkins.cd.test.fabric8.io/che-version-updater/notify
+    # every time a new version of Che is available
+    PROD_IMAGE_DOCKER_HUB="rhche/rh-che-server:${CHE_SERVER_DOCKER_IMAGE_TAG}"
```

Comment:
@l0rd @davidfestal I a bit got lost with `https://jenkins.cd.test.fabric8.io/che-version-updater/` do we really need this ?